DetDescr/src/DetDescr/EcalHexReadout.cxx

```diff
@@ -20,8 +21,11 @@ EcalHexReadout::EcalHexReadout(const framework::config::Parameters& ps)
   nCellRHeight_ = ps.getParameter<double>("nCellRHeight");
   verbose_ = ps.getParameter<int>("verbose");
 
+  if (nCellRHeight_ == 35) rotated_ = true;
+  else rotated_ = false;
```

Comment:
`rotated_` should be it's own parameter that is decoupled from `nCellRHeight_`.

Also sidenote, I think "rotated" is too vague. Maybe `pointUp`? (That's not great either....) Something that says "when this is true, the module has a vertex at the top and when its false, the module has a flat side up".

Reply:
Tried to come up with a better name that isn't excessively long -- `cornersSideUp` at least suggests what we're going for but perhaps is not clear enough? There is of course an explanatory comment so hopefully that helps.